ui/lib/core/addon/mixins/replication-actions.js

```diff
@@ -11,7 +11,6 @@ import { task } from 'ember-concurrency';
 
 export default Mixin.create({
   store: service(),
-  router: service(),
```

Comment:
I wonder why we had this service in here to begin with🤔

Reply:
Yeah, I was going to move the service to the components that used the `ReplicationActions` mixin but then realized this wasn't used anywhere in the mixin or where the mixin was used?? My guess is left behind during a refactor?

Reply:
But would definitely appreciate a second set of eyes double checking!